jest.config.js

```diff
@@ -10,6 +10,19 @@ module.exports = {
         "<rootDir>/node_modules/",
       ],
     },
+    {
+      displayName: "jest-eslint",
+      runner: "jest-runner-eslint",
+      testMatch: [
+        "<rootDir>/src/base/static/components/**/*.js",
+        "<rootDir>/src/base/static/utils/**/*.js",
+        "<rootDir>/scripts/**/*.js",
+      ],
+      testPathIgnorePatterns: [
+        "<rootDir>/src/base/static/libs/",
```

Comment:
Note that this will have to be updated to `static-libs` when we merge with the map port branch.